### ISSUE TITLE
New version can't compile

### ISSUE THREAD

**Issue by RamsesMartinez** [lusaxweb/vuesax#821]
After generating the project with the latest version, I get the following error:

```shell
These dependencies were not found:
* vuesax in ./src/plugins/vuesax.js
* vuesax/dist/vuesax.css in ./src/plugins/vuesax.js

To install them, you can run: npm install --save vuesax vuesax/dist/vuesax.css
```

When I check the vuesax folder in my node_modules folder, I found that the **/dist** folder does not exist

![image](https://user-images.githubusercontent.com/10599008/81511345-4e2a2000-92de-11ea-80e3-9d166278208e.png)

I also deleted the node modules folder and recompiled it and get the same error.

At the moment I returned to version 3.11.13.

- Debian 10
- Node.js v12.16.3
- 3.11.14
- Yarn

**Comment by RamsesMartinez** [comment]
Fixed in https://github.com/lusaxweb/vuesax/commit/b857a2301cdabcd71a955430f914c218322f68b0

**Comment by wasinr** [comment]
@RamsesMartinez How can you fix this issue.

**Comment by RamsesMartinez** [comment]
> @RamsesMartinez How can you fix this issue.

It was fixed in 3.11.16 version
